network_simulator/__init__.py

```diff
@@ -1,7 +1,9 @@
-"""Initialization file for the package network_simulator.
+"""
+Initialization file for the package network_simulator.
```

Comment:
It would be useful to clarify that initializer.py is the actual entry-point into the simulation. I don't think we'll be using __init__.py at all. In any case, if we want to use this for PyPI, the submodules should be defined as shown here: https://docs.python.org/2/tutorial/modules.html#importing-from-a-package. Otherwise, "from network_simulator import *" does not work.